aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/tests/QueryInstrumentationTest.java

```diff
@@ -156,148 +156,6 @@ public void testQueryPostsWithUserPoolsAuthorization() {
         }
     }
 
-    @Test
-    public void testCRUDWithSingleClient() {
-        List<AWSAppSyncClient> clients = new ArrayList<>();
-        clients.add(AWSAppSyncClients.withAPIKEYFromAWSConfiguration(false, REASONABLE_WAIT_TIME_MS));
-        PostCruds.test(clients);
```

Comment:
Are we able to delete the `PostCruds` too?

Reply:
I'm refactoring the imports. I should be able to get rid of it